examples/email_report.py

```diff
@@ -0,0 +1,115 @@
+"""Script that sends an email report with the 10 most important CVEs and some of there attributes"""
+import smtplib
+import ssl
+from collections import defaultdict
+from datetime import date, datetime
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+from cbw_api_toolbox.cbw_api import CBWApi
+
+# SMTP Configuration
+SENDER_EMAIL = "noreply@cyberwatch.fr"
+RECEIVER_EMAILS = ["", ""]
+LOGIN_SMTP = ""
+PASSWORD_SMTP = ""
+SMTP_SERVER = ""
+PORT_SMTP = 465
+
+# API Configuration
+API_KEY = ""
+SECRET_KEY = ""
+API_URL = ""
+CLIENT = CBWApi(API_URL, API_KEY, SECRET_KEY)
+
+# Filters
+MIN_SCORE = 8
+
+
+def sort_by_keys(count, score, exploit_available):
+    """
+        Return formatted string with digits to be used by sorted() to sort by keys
+    """
+    format_sort = "{}_".format(
+        exploit_available[0]) + "{0:0=4d}".format(int(score)) + "_{0:0=4d}".format(count)
+    return format_sort
+
+
+CVE_COUNTS = defaultdict(int)
+CVE_LIST = {}
+
+for server_item in CLIENT.servers():
+    server = CLIENT.server(server_item.id)
+    if server.cve_announcements is not None:
+        for cve_announcement in server.cve_announcements:
+            if cve_announcement.cve_score is not None:
+                CVE_COUNTS[cve_announcement.cve_code] += 1
+                if cve_announcement.cve_score >= MIN_SCORE:
+                    CVE_LIST[cve_announcement.cve_code] = {
+                        "count": CVE_COUNTS[cve_announcement.cve_code],
+                        "cve_score": cve_announcement.cve_score,
+                        "published": cve_announcement.published
+                    }
+
+GET_PUBLIC_EXPLOIT = {}
+for cve_code in CVE_LIST:
+    cve = CLIENT.cve_announcement(cve_code)
+    GET_PUBLIC_EXPLOIT[cve_code] = {
+        "count": CVE_LIST[cve_code]["count"],
+        "exploitable": str(cve.exploitable),
+        "cve_score": CVE_LIST[cve_code]["cve_score"],
+        "published": datetime.strptime(CVE_LIST[cve_code]["published"][:10], "%Y-%m-%d")}
```

Comment:
What does the part above do?

Reply:
I think it should be merged with
'''python
                    CVE_LIST[cve_announcement.cve_code] = {
                        "count": CVE_COUNTS[cve_announcement.cve_code],
                        "cve_score": cve_announcement.cve_score,
                        "published": cve_announcement.published
                    }
'''